application/application.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/tidepool-org/platform/version"
 )
 
+// TEST: Optionally force Local time zone to UTC during local development
```

Comment:
Can't do a flag since this can only be done in the `init` function. The time.Local and time.UTC are true global variables and are not protected by `sync`. If you try to modify them outside of an `init` function then it is picked up as a potential race condition. 

Besides, this is really only for a certain type of debugging. I wouldn't want it enabled for all local or development usage. I've used it for helping me debug certain UTC time formatting issues. The comment is really just a reminder to me how to switch over when necessary, which isn't all that often. The one or two times I've need to force it, I've had to figure it out all over again. With the comment I don't have to. :)